shared/client/client.go

```diff
@@ -110,9 +119,17 @@ func (c *HappyClient) Do(req *http.Request) (*http.Response, error) {
 func (c *HappyClient) addAuth(req *http.Request) error {
 	token, err := c.tokenProvider.GetToken()
 	if err != nil {
-		return errors.Wrap(err, "failed to get token")
+		return errors.Wrap(err, "failed to get oidc token")
 	}
 	req.Header.Add("Authorization", fmt.Sprintf("Bearer %s", token))
 
+	creds, err := c.awsCredProvider.GetCredentials(context.Background())
```

Comment:
I think this is also a place you need to pass the context in instead of using context.Background

Reply:
I think I'll leave this how it is for now since the client doesn't have a context being passed to it currently. We can thread that through later if necessary